pytapo/__init__.py

```diff
@@ -563,7 +563,13 @@ def performRequest(self, requestData, loginRetryCount=0):
             encryptedResponse = responseData["result"]["response"]
             encryptedResponse = base64.b64decode(responseData["result"]["response"])
             try:
-                responseJSON = json.loads(self.decryptResponse(encryptedResponse))
+                decodedResponse = self.decryptResponse(encryptedResponse).decode("utf-8")
+                
+                # Check if the response is wrapped with double curly braces
+                if decodedResponse.startswith('{{') and decodedResponse.endswith('}}'):
+                  decodedResponse = decodedResponse.replace('{{', '{').replace('}}', '}')
```

Comment:
Calling replace would replace `{{` and `}}` inside as well, which is not what we want. 
I suggest just trimming 1 character from the start and one from the end.

Reply:
good catch

Reply:
I don't know python, so I would be glad if you could modify it further

Reply:
Put this in please on line 570 instead of what is there now:

`decodedResponse = decodedResponse[1:-1]`

Reply:
Hopefully this fix will not be needed though and we can get the hub to respond as expected, see https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/455#issuecomment-1817484699 lets continue there